pkg/registries/factory_impl.go

```diff
@@ -33,7 +33,9 @@ func (e *factoryImpl) CreateRegistry(source *storage.ImageIntegration, options .
 	var creator types.Creator
 	var exists bool
 
-	if !source.Autogenerated || disableRepoListForAll {
+	// The source != nil condition here ensures that sourced integrations (ie: with namespace and secret name)
+	// do not use repo lists for matching, these integrations will be matched elsewhere based on source.
```

Comment:
where is the "elsewhere"?

Reply:
For sourced autogenerated, would be during enrichment [here](https://github.com/stackrox/stackrox/blob/9ea85f3120e80388928c766c350010bb52175311/pkg/images/enricher/enricher_impl.go#L898)